mdata/notifierKafka/notifierKafka.go

```diff
@@ -26,40 +26,43 @@ type ClKafka struct {
 	StopChan  chan int
 	// signal to PartitionConsumers to shutdown
 	stopConsuming chan struct{}
-	Cl
+	mdata.Notifier
 }
 
-func NewKafka(instance string, metrics Metrics, stats met.Backend) *ClKafka {
-	client, err := sarama.NewClient(cfg.Brokers, cfg.Config)
+func NewNotifierKafka(instance string, metrics mdata.Metrics, stats met.Backend) *NotifierKafka {
+	messagesPublished = stats.NewCount("cluster.messages-published")
+	messagesSize = stats.NewMeter("cluster.message_size", 0)
```

Comment:
now the notifier plugins have their own copies of the metrics, but with the same keys. because it ultimately gets sent to statsd (for now) this doesn't really matter but I suspect this is not what was intended. we should probably add the notifier type into the metrics instead. (e.g. `notifier.kafka.messages-published`)